cpp/src/reshape/stack.cu

```diff
@@ -1,109 +0,0 @@
-/*
```

Comment:
Where's the new implementation?

Reply:
Per the transition guidelines, there isn't one until the declarations get reviewed.